cookbook/jackrabbit.rst

```diff
@@ -57,6 +57,16 @@ can e.g. be set in your ``.env`` file.
     The ``PHPCR_WORKSPACE`` is something similar as a database name so it is best practice
     to have a similar value for it, for example: ``su_myproject`` in your ``.env`` files.
 
+    The ``JACKRABBIT_URL`` by default is running under `http://127.0.0.1:8080/server/`
+    or `http://127.0.0.1:8080/jackrabbit/server/` depending on Version and OS.
```

Comment:
```suggestion
```suggestion
    The ``JACKRABBIT_URL`` needs to point to your jackrabbit backend. 
    Depending on your OS and jackrabbit version, the default should be `http://127.0.0.1:8080/server/`
    or `http://127.0.0.1:8080/jackrabbit/server/`.
```
```